gazebo_ros/src/gazebo_ros_init.cpp

```diff
@@ -176,6 +176,10 @@ void GazeboRosInit::Load(int argc, char ** argv)
     rclcpp::ParameterValue(GazeboRosInitPrivate::DEFAULT_PUBLISH_FREQUENCY));
   impl_->throttler_ = Throttler(rate_param.get<double>());
 
+  // PerformanceMetrics parameter
+  impl_->ros_node_->declare_parameter<bool>("enable_performance_metrics", true);
+
+
```

Comment:
nitpick: remove this extra blank line (I thought a linter might complain, but perhaps not).

Reply:
Done